### ISSUE TITLE
Operator OSMS disable feature added to v3.0.1, operator module which is being used also needs to be updated from v3.0.0 to v3.0.1 

### ISSUE THREAD

**Issue by KSN2510** [oracle-terraform-modules/terraform-oci-oke#443]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. 
# Please remove any sensitive information from configuration files before sharing them. 
```

### Steps to Reproduce

1. `terraform apply`

**Comment by KSN2510** [comment]
Here is the PR which has version change
[](https://github.com/oracle-terraform-modules/terraform-oci-oke/pull/444)

**Comment by hyder** [comment]
fixed in #444
